client/src/common/utils/events.js

```diff
@@ -0,0 +1,12 @@
+export const attachBeforeUnloadEvent = callback =>
+  window.addEventListener('beforeunload', callback);
```

Comment:
hmm, if we use sth like this:
```
export const attachBeforeUnloadEvent = (callback = handleWindowBeforeUnload) =>
  window.addEventListener('beforeunload', callback);
```
there will be node need to import `handleWindowBeforeUnload` in ever file where we want to add/remove onBeforeUnload event, but we will be able to pass different callback;

Reply:
@Ciunkos  @marekrozmus  what do you think?